server/config/topic_page/equity/US_Places.textproto

```diff
@@ -35,6 +35,265 @@ metadata {
+        stat_var_key: "max_temp_prediction"
+      }
+      tiles {
+        title: "Disabilities (${yDate}) vs. Predicted Max Temperature (${xDate})"
+        type: SCATTER
+        stat_var_key: "disabilities_prevalence"
+        stat_var_key: "max_temp_prediction"
+      }
+    }
+    columns {
+      tiles {
+        title: "Coronary Heart Disease Prevalence (${yDate}) vs. Predicted Max Temperature (${xDate})"
+        type: SCATTER
+        stat_var_key: "heart_disease_prevalence"
+        stat_var_key: "max_temp_prediction"
+      }
+      tiles {
+        title: "No Health Insurance (${yDate}) vs. Lifetime Cancer Risk due to Air Pollutants (${xDate})"
+        type: SCATTER
+        stat_var_key: "no_health_insurance"
+        stat_var_key: "cancer_risk"
+      }
+    }
+  }
+
+  blocks {
+    title: "Demographics"
+    columns {
+      tiles {
+        title: "Poverty Rate (${yDate}) vs. PM 2.5 Emissions (${xDate})"
+        type: SCATTER
+        stat_var_key: "poverty_rate"
+        stat_var_key: "PM25_emissions"
+      }
+      tiles {
+        title: "Per Capita Asian Population (${yDate}) vs. Lifetime Cancer Risk due to Air Pollutants (${xDate})"
+        type: SCATTER
+        stat_var_key: "asian_population"
+        stat_var_key: "PM25_emissions"
+      }
+      tiles {
+        title: "Per Capita Black or African American Population (${yDate}) vs. PM 2.5 Emissions (${xDate})"
+        type: SCATTER
+        stat_var_key: "black_population"
+        stat_var_key: "PM25_emissions"
+      }
+    }
+    columns {
+      tiles {
+        title: "Household Median Income (${yDate}) vs. Lifetime Cancer Risk due to Air Pollutants (${xDate})"
+        type: SCATTER
+        stat_var_key: "household_median_income"
+        stat_var_key: "cancer_risk"
+      }
+      tiles {
+        title: "Per Capita Hispanic Population (${yDate}) vs. Ozone Emissions (${xDate})"
+        type: SCATTER
+        stat_var_key: "hispanic_population"
+        stat_var_key: "O3_emissions"
+      }
+      tiles {
+        title: "Per Capita American Indian or Alaskan Native Population (${yDate}) vs. PM 2.5 Emissions (${xDate})"
+        type: SCATTER
+        stat_var_key: "american_indian_population"
+        stat_var_key: "PM25_emissions"
+      }
+    }
+  }
+
+  blocks {
+    title: "Employment"
+    columns {
+      tiles {
+        title: "Unemployement Rate (${yDate}) vs. Predicted Max Temperature (${xDate})"
+        type: SCATTER
+        stat_var_key: "unemployment_rate"
+        stat_var_key: "max_temp_prediction"
+      }
+    }
+  }
+
+  stat_var_spec {
+    key: "CO2_emissions"
+    value {
+      stat_var: "Annual_Emissions_CarbonDioxide_NonBiogenic"
+      name: "CO2 Emissions"
+    }
+  }
+
+  stat_var_spec {
+    key: "max_temp_prediction"
+    value {
+      stat_var: "MaxTemp_Daily_Hist_5PctProb_Greater_Atleast1DayADecade_CMIP6_Ensemble_SSP585"
+      name: "MaxTemp Ensemble GaussianMixture 5PctProb Greater Atleast1DayAYear CMIP6 SSP245"
```

Comment:
Ohh, this is incorrect.  Lets do:

"Projected max temp (5% probability, decadal, SSP245)"